.gitlab-ci.yml

```diff
@@ -87,3 +88,18 @@ test_nearlib_release:
     # the PR is going to merge with. Before fix in bot, keep different version of .gitlab-ci.yml 
     # in staging and master.
     # - master 
+
+release_build:
+    stage: release
+    tags:
+    - shell
+    before_script:
+    - *setup_cache
+    script:
+    - cargo build -p near
```

Comment:
Why is non-release build needed?

Reply:
Alex need it in download to each remote node and run pytest